common/src/sql/dbinit.sql

```diff
@@ -128,6 +128,39 @@ CREATE INDEX ON omicron.public.service (
     sled_id
 );
 
+-- x509 certificates which may be used by services
+CREATE TABLE omicron.public.certificate (
+    -- Identity metadata (asset)
```

Comment:
nit: this doesn't look like asset metadata.  it looks like resource metadata, since it has a name, mtime, etc.

Reply:
You're right; this was a typo. I started this PR making it "internal only", but when I realized it made sense as part of the API, I added the "name" and other fields. Updated this comment in a4972fd9893d9457c9a78a0d7dc9c1494116de97